vimms/MassSpec.py

```diff
@@ -102,6 +102,101 @@ def __init__(self, chem, query_rt, ms_level, peaks, scan_id):
     def __repr__(self):
         return 'MS%d FragmentationEvent for %s at %f' % (self.ms_level, self.chem, self.query_rt)
 
+
+class TaskManager(object):
```

Comment:
Ideally the TaskManger would manage the lists itself, but for the sake of getting experiments done, this should be okay.

Reply:
Thanks. Once we confirm this run, I could move higher-level functions into the task manager to manage itself.